cloudify/amqp_client.py

```diff
@@ -374,16 +389,24 @@ def publish(self, message, **kwargs):
         })
 
 
-class _RequestResponseHandlerBase(TaskConsumer):
-    def __init__(self, exchange):
-        super(_RequestResponseHandlerBase, self).__init__(exchange)
-        self.queue = None
-
-    def _register_queue(self, channel):
+class _RequestResponseHandlerBase(object):
+    def __init__(self, exchange, routing_key='', *args, **kwargs):
+        super(_RequestResponseHandlerBase, self).__init__(*args, **kwargs)
```

Comment:
Let's add a comment here re: this super call, so that posterity won't try to remove it as unnecessary.